library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -435,6 +435,7 @@ public Iterator<Vertex> vertices(final Object... vertexIds) {
                 .map(generator::_apply)
                 .filter(Vertex.class::isInstance)
                 .map(e -> (Vertex) e)
+                .limit(variables.getAllElementsLimit())
```

Comment:
If this is now being applied to all 'gets' should it be rebranded as a `elementLimit` rather than a `getAllElementsLimit` and also be applied in the actual gaffer operation for a normal `GetElements` operation?